numpy/f2py/rules.py

```diff
@@ -1276,16 +1276,16 @@ def buildmodule(m, um):
     fn = os.path.join(options['buildpath'], vrd['coutput'])
     ret['csrc'] = fn
     with open(fn, 'w') as f:
-        f.write(ar['modulebody'].replace('\t', 2 * ' '))
-    outmess('\tWrote C/API module "%s" to file "%s"\n' % (m['name'], fn))
+        f.write(ar['modulebody'].replace('    ', 2 * ' '))
```

Comment:
This part, which I assume leads to the doc change need, is confusing me slightly.  Anyway, I am happy with this, and OK with just putting this in as a step in the right direction in any case.  Are Pearu or Melissa following these f2py issues?

Reply:
Yup, though this one isn't directly linked to an issue, I see that that was a rather ham handed change, so I rolled back that part (just in case any downstream `f2py` projects are checking docstrings...)